### PR TITLE
Add column dnd example

### PR DESCRIPTION
**Pull request by peraltafederico**
This pull request adds a column drag and drop implementation example by using `react-dnd`. In the example I set the table width to 100%  and I only use table elements, so no layout plugin hook is required.

**EDIT: Here it is a Codesandbox with the implementation https://codesandbox.io/s/react-table-column-dnd-hkj2y** :rocket:

### CODE REVIEW

File: examples/column-dnd/src/DraggableHeader.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import { getEmptyImage } from 'react-dnd-html5-backend'
+import ItemTypes from './itemTypes'
+import styled from 'styled-components'
+
+const StyledTh = styled.th`
+  cursor: move;
+  opacity: ${props => (props.isDragging ? 0 : 1)};
+`
+
+const DraggableHeader = ({ column, index, reoder }) => {
```
Comment (bugzpodder) [inline examples/column-dnd/src/DraggableHeader.js:12]: reorder

Comment (peraltafederico) [reply examples/column-dnd/src/DraggableHeader.js:12]: Hi @bugzpodder, is something wrong with `reorder`? Maybe the prop name?

Comment (bugzpodder) [reply examples/column-dnd/src/DraggableHeader.js:12]: reoder vs reorder

Comment (peraltafederico) [reply examples/column-dnd/src/DraggableHeader.js]: @bugzpodder Incredibly that I didn't see it earlier! 😆 Fixed in commit 224d141dfa4d0a0623348dfb37d90ada1c5eb08e

**Comment by zfha** [comment]
when to merge to master ?

**Comment by luismasg** [comment]
This works pretty good !!
thanks for the example.
why do we need the scrolling component ?

**Comment by richbachman** [comment]
@peraltafederico Is there a codesandbox available for this one?

**Comment by fperaltalitebox** [comment]
> @peraltafederico Is there a codesandbox available for this one?

I'm gonna create it today at night!

**Comment by fperaltalitebox** [comment]
> This works pretty good !! thanks for the example. why do we need the scrolling component ?

@luismasg  Because if there's a lot of rows and columns that doesn't enter in the screen and you want to drag a column to the right we'll need an auto scroll behavior. Today I'm gonna create a codesandbox with it so you can test it! :)

**Comment by peraltafederico** [comment]
> > @peraltafederico Is there a codesandbox available for this one?
> 
> @richbachman I'm gonna create it today at night!

Hey @richbachman Here it is the Codesandbox with the working example! https://codesandbox.io/s/react-table-column-dnd-hkj2y

cc @luismasg

**Pull request closed without merging**